### PR TITLE
fix: Enabling role=button for multiselect widgets

### PR DESCRIPTION
**Pull request by poojari-venugopal**
This commit enables the role="button" attribute on MultiSelect widget, so that MultiSelect icons will be included in the list of
buttons displayed by screen readers and become focusable by tab.

### CODE REVIEW

File: src/aria/widgets/form/MultiSelect.js [hunk 1]
```diff
@@ -85,6 +86,18 @@ module.exports = Aria.classDefinition({
...
+        var tabIndex = isWaiAria ? (cfg.tabIndex != null ? this._calculateTabIndex() : "0") : "-1";
+        this._iconsAttributes = {
+            "dropdown": 'tabindex="' + tabIndex + '"' + iconTooltip
+        };
+
+        if (isWaiAria) {
+            var waiIconLabel = cfg.waiIconLabel;
+            this._iconsAttributes.dropdown += ' role="button" aria-expanded="false" aria-haspopup="true"  ' +
+               (waiIconLabel ? 'aria-label="' + waiIconLabel + '" ' : "");
+        }
```
Comment (divdavem) [inline src/aria/widgets/form/MultiSelect.js:100]: Why do you need this?
Isn't the change in `DropDownTextInput` (adding role="button") sufficient?

File: src/aria/widgets/form/MultiSelect.js [hunk 2]
```diff
@@ -167,7 +180,7 @@ module.exports = Aria.classDefinition({
         _toggleDropdown : function () {
             this._updateFocusNoKeyboard(true);
             if (!this._hasFocus) {
-                this.focus(null, true);
+                this.focus();
```
Comment (divdavem) [inline src/aria/widgets/form/MultiSelect.js:183]: Why is this change necessary?

File: src/aria/widgets/form/DropDownTextInput.js [hunk 3]
```diff
@@ -41,7 +41,7 @@ module.exports = Aria.classDefinition({
             "dropdown": iconTooltip
         };
         if (cfg.waiAria) {
-            this._iconsAttributes.dropdown += ' aria-expanded="false" aria-haspopup="true"';
+            this._iconsAttributes.dropdown += 'role="button" aria-expanded="false" aria-haspopup="true"';
```
Comment (divdavem) [inline src/aria/widgets/form/DropDownTextInput.js:44]:
```suggestion
            this._iconsAttributes.dropdown += ' role="button" aria-expanded="false" aria-haspopup="true"';
```

**Comment by divdavem** [comment]
Have you fully tested the change in the application? Is it working as expected?

**Comment by poojari-venugopal** [comment]
> Have you fully tested the change in the application? Is it working as expected?

Yes David, tested it in local, and its working
